test/tests/morebits.wikitext.js

```diff
@@ -0,0 +1,41 @@
+QUnit.module('Morebits.wikitext');
+QUnit.test('parseTemplate', assert => {
+	var makeTemplate = function(data) {
+		var template = '{{' + data.name;
+		Object.keys(data.parameters).forEach(function(key) {
+			// Necessary until fixing a bug in parseTemplate
```

Comment:
#1214

Reply:
You can write the test for the ideal behaviour here and disable it using `QUnit.skip`, to be enabled later. Writing flawed tests just to get them to pass defeats the point of testing.